CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.39.120](https://github.com/brave/brave-browser/releases/tag/v1.39.120)
+
+ - Changed position of VPN toggle under settings menu. ([#23240](https://github.com/brave/brave-browser/issues/23240))
+ - Fixed Brave Firewall + VPN crash when device attempts connecting to Google Play Store after losing network connectivity in certain cases. ([#23177](https://github.com/brave/brave-browser/issues/23177))
```

Comment:
Maybe change to browser crash rather than `Brave Firewall + VPN` ? as its the app crashing not just that feature.

Reply:
Done 👍